.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: c
+sudo: required
+dist: trusty
+
+install:
+        - sudo apt-add-repository -y ppa:team-gcc-arm-embedded/ppa
+        - sudo apt-get update
+        - sudo apt-get install -yy gcc-arm-embedded
+
+script:
+        - cd efm32hg-blinky
+        - make deps
```

Comment:
I think we should open a ticket to do that work (or at least print a warning).

Travis doesn't give us access to the build artefacts, so think of it as a smoke test and less so creating something that boots.

Once I get my qemu model working we will boot the firmware in the model, so we need it working by then.